tasksCalendar/view.js

```diff
@@ -124,6 +124,11 @@ function getIcon(task) {
 	if (icon) { return icon } else { return "" };
 };
 
+function getTextColor(task) {
```

Comment:
3 different functions to get a value from a key inside note-meta is not so good. Here I write a single function to get each meta-value by key.

Reply:
I see, thank you for your time and all comments :)